src/app/components/activity/feed/devlog-post/text/text.vue

```diff
@@ -8,12 +8,14 @@
 		}"
 	>
 		<template v-if="isOpen">
-			<div class="page-cut" />
+			<div class="page-cut">
+				<div class="page-cut-content-placeholder" />
```

Comment:
won't this show an overlay on top of the border as a cutout? why do we need this?

Reply:
also, hyphenate

Reply:
The placeholder is just a transparent div with the same height as the `page-cut-content` class. This aligns the top page-cut when the article expands and the article expansion control button is moved to the bottom.

Reply:
I mean that it prevents shifting of the page-cut when the article expands. Otherwise the page-cut would shift up when it loses the button child.